app/main/views.py

```diff
@@ -185,6 +196,12 @@ def get_service_by_id(service_id):
 
 
 @main.route('/search')
+@deprecated(dies_at=datetime(2016, 1, 1))
+def redirect_search():
+    return redirect(url_for(".search", **request.args), code=301)
```

Comment:
@pcraig3 `.args` only contains GET parameters. There's also `request.form` for form POST params and `request.values` with the combined GET and POST parameters. But you probably only want GET for redirects.